contrib/seeds/nodes_main.txt

```diff
@@ -650,518 +650,6 @@
 [2a0f:df00:0:254::46]:8333
 [2c0f:f598:5:1:1001::1]:8333
 [2c0f:fce8:0:400:b7c::1]:8333
-226eupdnaouu4h2v.onion:8333
```

Comment:
Please also run `cd contrib/seeds && python3 generate-seeds.py . > ../../src/chainparamsseeds.h` to update `chainparamsseeds.h` (in the same commit?).

Reply:
Thanks! Done.